src/pytorch_lightning/utilities/seed.py

```diff
@@ -153,3 +151,8 @@ def isolate_rng() -> Generator[None, None, None]:
     states = _collect_rng_states()
     yield
     _set_rng_states(states)
+
+
+def _log_info(message: str) -> None:
```

Comment:
We don't need a helper function here, right? log.info can be called directly in the seed function above IMO

Reply:
I used this trick for easy testing. Without helper function I have to know SEED value to generate expected message to log.info and if I use None as SEED in `seed_everything` function it generates randomly. So, I used the helper function to check logic of adding rank info despite logging message.

Reply:
If you want to add a test for that, it should be easy to mock this. You can mock directly the log.info method and intercept the message that goes in.